### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by abdullah41-tech** [Cyfrin/foundry-full-course-cu#1765] [General]
Hey guys! Giving myself a pat on the back because I'm finally getting into Blockchain. It's never too late to start something new and dive into new things with some positive vibes, right? Wish me luck! Any tips?

**Comment by EngrPips** [comment]
Welcome aboard, Abdullah. I wish you the best of luck in your journey. My only advice is to not give up!

**Reply by abdullah41-tech** [reply]
Hey man! Nice connecting with you. Tbh, that is the great advice i'd be needing down the road. Thanks, appreciate it.

**Reply by EngrPips** [reply]
Anytime, friend. Keep crushing it.

**Comment by urosognjenovic** [comment]
Read a lot, engage with the community, use the products yourself and try to make them as well and you will gain a lot of knowledge.

**Reply by abdullah41-tech** [reply]
Hey braveman! Nice connecting with you buddy. I will put those tips in the account and start working on them gradually. Thanks, I appreciate it.

**Comment by helloram007** [comment]
Hi, This is Ramchandhar Rapolu, i have just completed Bloakchain basics and moving into Solidity 101.

**Reply by EngrPips** [reply]
Congratulations on your progress, @helloram007, Keep going; Don't stop

**Comment by helloram007** [comment]
Just got my first Contract deployed. Kudos and a pat on my back :)))))

**Reply by EngrPips** [reply]
Awesome, Keep going don't stop.

**Comment by davidpopovici01** [comment]
Currently refreshing my knowledge on this topic and these materials are great!

**Reply by EngrPips** [reply]
Excellent, revision is the key to solid knowledge.

**Comment by helloram007** [comment]
I am really enjoying the tutorials, they are crisp, clear and concise.
Thanks Patrick.

**Reply by EngrPips** [reply]
Patrick is doing the most.
